### DISCUSSION TITLE
I am not able to re-verify my student status

### DISCUSSION THREAD

**Discussion by reuseman** [community/community#41903] [GitHub Education]
Hey GitHub Team,

I previously had the GitHub student pack, but in the last week I moved to a different country, and I will stay here just one week. But now I cannot have the access to my pack anymore and I am not able to renew it.

![image](https://user-images.githubusercontent.com/17575915/208088842-ac4194bc-99e4-41e5-8b00-8e8c779bc11f.png)

What can I do? 

Thanks

**Comment by samus-aran** [comment]
Hi @reuseman 👋 

Occasionally it is necessary for us to ask you to reverify your current academic status in order to continue receiving your GitHub Education benefits. We may require this when seemingly-legitimate applicants have submitted academic affiliation proof which has been discovered to be used by bad actors as well.

There are a number of rejection reasons detailed in your screenshot. Thank you for supplying them. Let's see if we can assist.

It looks like your school issues the coupons to its students and faculty via the GitHub Campus program. This means you’ll have to talk to the GitHub administrator for your school and request that they assist you with your Global Campus benefits.

Since you state that you are in another country for a limited time it would be suggested that you re-apply when you return.

**Comment by obaidsafi51** [comment]
I want to reverify my account for student developer pack but I am getting problem. How can I solve this.
![Screenshot (56)](https://github.com/community/community/assets/134671014/ddc5de0b-af6f-49c5-b585-916b53c3df01)

**Comment by ZapeeoSheikh** [comment]
Hi @samus-aran !!

I want to re-verify my documents for github developer pack. I am not able to perform re-verification. They are saying please submit another application. But my student mail is already attached with my github account now when i am selecting my student mail, it popped up with already registered. Now i am neither able to re-submit my docs nor able to apply again with the same mail.

Please resolve this issue.
